apim/Initialize-ApimDeveloperPortal.ps1

```diff
@@ -32,6 +33,36 @@ param(
     [string]$ApimDeveloperPortalCustomDomainSuffix
 )
 
+function Remove-ApimProduct {
+    param (
+        #Default APIM Product
+        [Parameter(Mandatory = $true)]
+        [ValidateSet("starter", "unlimited")]
+        [string]$Product
+    )
+
+    # --- Get subscription of Product
+    $GetProductSubscriptionParameters =
+    "--method", "get",
+    "--uri", "https://management.azure.com/subscriptions/$($AzureSubscriptionId)/resourceGroups/$($ApimResourceGroupName)/providers/Microsoft.ApiManagement/service/$($ApimName)/products/$($Product)/subscriptions?api-version=2019-12-01"
```

Comment:
Do you need the double $($AzureSubscriptionId)? Can it just be $(AzureSubscriptionId)

Reply:
Could just be $AzureSubscriptionId but I think it makes it stand out more that it's a variable, and consistent with other variable references in the script